website/content/docs/k8s/connect/ingress-controllers.mdx

```diff
@@ -6,16 +6,15 @@ description: Configuring Ingress Controllers With Consul On Kubernetes
 
 # Configuring Ingress Controllers with Consul on Kubernetes
 
--> This topic requires familiarity with [Ingress Controllers](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/)
+-> This topic requires familiarity with [Ingress Controllers](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/).
+  The following information requires Consul 1.10+, Consul-k8s 0.26+, Consul-helm 0.32+ configured with
+  [Transparent Proxy](/docs/connect/transparent-proxy) mode enabled.
 
--> The following information requires Consul 1.10+, Consul-k8s 0.26+, Consul-helm 0.32+ configured with
-[Transparent Proxy](/docs/connect/transparent-proxy) mode enabled.
+~> If you are looking for a fully supported solution for ingress traffic into Consul Service Mesh, please visit [Consul API Gateway](/docs/api-gateway) for instruction on how to install Consul API Gateway along with Consul on Kubernetes.
```

Comment:
```suggestion
If you are looking for a fully supported solution for ingress traffic into Consul Service Mesh, please visit [Consul API Gateway](/docs/api-gateway) for instruction on how to install Consul API Gateway along with Consul on Kubernetes.
```

Reply:
I personally think we should leave this as a warning, as the other solutions are not end to end supported.